### PR TITLE
adds neet ranks

### PR DESCRIPTION
**Pull request by suryabulusu**
- Adds NEET data

### CODE REVIEW

File: constants.js
Comment (pritamps) [inline constants.js]: Remove extra space between the end and the `""`. Messing up alignment

File: constants.js
Comment (pritamps) [inline constants.js]: ^

File: pages/college_predictor.js
Comment (pritamps) [inline pages/college_predictor.js]: Remove this line. No need to comment :)

File: pages/college_predictor.js
Comment (pritamps) [inline pages/college_predictor.js]: What is the comment for?

File: pages/college_predictor.js
Comment (pritamps) [inline pages/college_predictor.js:24]: Remove `item` from all the variable names. Doesn't really add much

File: pages/college_predictor.js
Comment (pritamps) [inline pages/college_predictor.js]: Replace with checking if exam is main or advanced. Clearer for readers and as we add more exams it will make sense

File: pages/college_predictor.js
Comment (pritamps) [inline pages/college_predictor.js:28]: These variables are not important for `NEET`?

File: pages/college_predictor.js
Comment (pritamps) [inline pages/college_predictor.js:49]: Use better variable names than `rankA` and `rankB` . Also, shouldn't we be sorting by closing rank?

File: pages/college_predictor.js
Comment (pritamps) [inline pages/college_predictor.js:52]: Why do we need a customized sort? If we're just sorting by the `"College Rank"` column - a simple sort won't work?

File: pages/college_predictor.js
Comment (pritamps) [inline pages/college_predictor.js:81]: `NEET` mein category rank is not important?

File: pages/index.js
Comment (pritamps) [inline pages/index.js:55]: Rather than this, add a different route for `NEET` so it's clearer

`/college_predictor?exam=NEET[or JEE]...`

**Review by pritamps — CHANGES_REQUESTED** [review]
Some comments!

File: pages/college_predictor.js
Comment (suryabulusu) [reply pages/college_predictor.js:28]: for now, yes. only relying on round number and category. i'll wait for review from ops team

File: pages/college_predictor.js
Comment (suryabulusu) [reply pages/college_predictor.js:49]: closing rank filtering happens in previous lines to obtain `filteredData`

within filtered colleges -- we display colleges with better college rank.
now, if college rank is same (ie, rows with same college but different branch) -- we sort using opening rank [that is, of the branch. typically CSE opening rank better than EE opening rank]

File: pages/college_predictor.js
Comment (suryabulusu) [reply pages/college_predictor.js:52]: you only wrote it :p let me check whats the best way to sort object data in js

**Pull request merged**
